backend/src/main/java/com/htecgroup/skynest/SkyNestApplication.java

```diff
@@ -22,4 +23,9 @@ public BCryptPasswordEncoder bCryptPasswordEncoder() {
   public ModelMapper modelMapper() {
     return new ModelMapper();
   }
+
+  @Bean
+  public UserValidator userValidator() {
```

Comment:
Instead of defining the Bean here, you can use **@Component** annotation on Class level, in this case in UserValidator class.

Reply:
I'll change it.